### PR TITLE
build: Remove eudev module

### LINKED ISSUE #88: Why does this app contain a copy of libudev.so?
This Flatpak contains copies of libudev.so.1.6.3 in /app/lib and /app/lib32 even though (current versions of) the Freedesktop runtimes already ship with libudev. This  breaks components of the runtime built against the runtime's newer version of libudev because the older version from the app gets picked up instead. Specifically this happened in the Lutris Flatpak: https://github.com/flathub/net.lutris.Lutris/issues/364

Is this dependency important? It looks like it was added over two years ago: https://github.com/flathub/org.winehq.Wine/commit/1e2a15f8f0203d3d18009fd112909e437db4b0a1 I tried removing it and the Flatpak still builds.

### LINKED ISSUE #583: Freedesktop 23.08 runtime problem with app starting
New Freedesktop 23.08 runtime released recently. After updating Minigalaxy won't start with new runtime. Not sure does this related to [this](https://github.com/sharkwouter/minigalaxy/issues/535) issue or not.  You can try Minigalaxy with new runtime in [flathub-beta](https://github.com/flathub/io.github.sharkwouter.Minigalaxy/tree/beta#minigalaxy-dev-version).

Снимок экрана от 2023-09-25 23-26-09

```
** (minigalaxy:2): WARNING **: 23:24:26.282: Failed to load shared library 'libwebkit2gtk-4.1.so.0' referenced by the typelib: /app/lib/libudev.so.1: version `LIBUDEV_247' not found (required by /usr/lib/x86_64-linux-gnu/libgudev-1.0.so.0)
```

### PR DESCRIPTION
**Pull request by tim77**
Fix: #88
Fix: https://github.com/sharkwouter/minigalaxy/issues/583

https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/master/NEWS#L320

### CODE REVIEW

File: org.winehq.Wine.yml [hunk 1]
```diff
@@ -149,39 +149,13 @@ modules:
           url-query: .assets[] | select(.name=="libusb-\($version).tar.bz2") | .browser_download_url
     modules:
```
Comment (gasinvein) [inline org.winehq.Wine.yml]: The `modules` property is now empty and should be removed.
```suggestion
```

File: org.winehq.Wine.yml [hunk 2]
```diff
@@ -149,39 +149,13 @@ modules:
...
-              version-query: $tag | sub("^[vV]"; "")
-              timestamp-query: .published_at
-              url-query: .assets[] | select(.name=="eudev-\($version).tar.gz") | .browser_download_url
-
   - name: libusb-32bit
     build-options:
       arch:
         x86_64: *compat_i386_opts
     sources: *libusb-sources
     modules:
```
Comment (gasinvein) [inline org.winehq.Wine.yml]: Same here, empty `modules`
```suggestion
```

**Review by gasinvein — CHANGES_REQUESTED** [review]

**Comment by fkrull** [comment]
Yeah this is the change I made as well. :+1:

**Comment by tim77** [comment]
I've tested this with Minigalaxy and this is solved my problem.

**Review by gasinvein — APPROVED** [review]
Thanks!

**Pull request merged**